openseespy-pip/openseespy/postprocessing/Get_Rendering.py

```diff
@@ -1448,3 +1376,254 @@ def update_plot(ii):
     plt.show()
     return line_ani
 
+
+
+
+
+# =============================================================================
+# Experimental Functions
+# =============================================================================
+
+def _plot_on_click(self, event):
+    
+    timeSteps = self.timeSteps
+    Disp_nodeArray =  self.Disp_nodeArray
+    self.DeflectedNodeCoordArray
+    
+    # Get Node
+    
+    # plot Nodes
+    
+    fig, ax = plt.subplots()
+    plt.plot(timeSteps)
+    
+    
+    
+    thisline = event.artist
+    xdata = thisline.get_xdata()
+    ydata = thisline.get_ydata()
+    ind = event.ind
+    points = tuple(zip(xdata[ind], ydata[ind]))
+    print('onpick points:', points)
+
+    
+class plot_deformedshape_Events:
+    
+    def __init__(self, Model="none", LoadCase="none", tstep = -1, scale = 10, overlap='no'):
+        self.Model = Model
+        self.LoadCase = LoadCase
+        self.tstep = tstep
+        self.scale = scale
+        self.overlap = overlap
+        
+        self.timeSteps, self.Disp_nodeArray = idbf._readNodeDispData(self.Model, self.LoadCase)
+        self.nodeArray, self.elementArray = idbf._readNodesandElements(self.Model)
+
+    def _plot_on_click(self, event):
+        
+        timeSteps = self.timeSteps
+        Disp_nodeArray =  self.Disp_nodeArray
+        DeflectedNodeCoordArray = self.DeflectedNodeCoordArray
+        
+        # Find the Node ID
+        thisline = event.artist
+        xdata = thisline.get_xdata()
+        ydata = thisline.get_ydata()
+        ind = event.ind[0]
+        fig, ax = plt.subplots()
+        # Disp = self.nodeArray[ind,1:], self.scale*Disp_nodeArray[:,ind,0]
+        line = ax.plot(timeSteps, self.scale*Disp_nodeArray[:,ind,0])
+        ax.set_xlabel('time (s)')
+        ax.set_ylabel('Displacement')
+        # print(ind)
+        # print(DeflectedNodeCoordArray[ind,:])
+        # points = tuple(zip(xdata[ind], ydata[ind]))
+        # print('onpick points:', points)
+
+    def plot(self):
+        """
```

Comment:
can we just call plot_deformedshape() function here instead of duplicating the code? Problems will arise if we change things in the plot_deformedshape() function.